CHANGES.md

```diff
@@ -66,6 +66,7 @@
 * Added Cloud Bigtable Provider extension to Beam SQL ([BEAM-11173](https://issues.apache.org/jira/browse/BEAM-11173), [BEAM-11373](https://issues.apache.org/jira/browse/BEAM-11373))
 * Added a schema provider for thrift data ([BEAM-11338](https://issues.apache.org/jira/browse/BEAM-11338))
 * Added combiner packing pipeline optimization to Dataflow runner. ([BEAM-10641](https://issues.apache.org/jira/browse/BEAM-10641))
+* Added support for thrift in KafkaTableProvider ([BEAM-11482](https://issues.apache.org/jira/browse/BEAM-11482))
```

Comment:
Unfortunately this will be in 2.28.0 since 2.27.0 was cut last week. Could you merge master (it should have the 2.28 section), and move this there?

Reply:
There's doesn't seem to be a section for `2.28.0` yet.
I was hoping we could get rid of all our custom thrift extensions in our project by just upgrading to `2.27.0`, but I wasn't fast enough. :)
Should I create the `2.28.0` section myself and add move this bullet over there?

Reply:
Ah yes please go ahead and create the section. Sorry I thought this happened as part of the release cut.

Reply:
https://github.com/apache/beam/pull/13610 just added the 2.28.0 section so you can merge or rebase to get it

Reply:
done